docs/how-to/install-pro-packages.md

```diff
@@ -0,0 +1,219 @@
+(install_pro_packages)=
+
+# Install Ubuntu Pro package slices
+
+This guide explains how to fetch and install {{ubuntu_pro}} package slices using
+Chisel.
+
+
+(install_pro_packages_pre_req)=
+
+## Pre-requisites
+
+You must have an {{ubuntu_pro}} subscription. See {{ubuntu_pro_docs}}.
+
+
+(install_pro_packages_configure_archives)=
+
+## Configure archive definitions
+
+To fetch and install Pro package slices, you need to define the Pro archives.
+Chisel uses the {ref}`chisel_yaml_format_spec_archives_pro` field to define a
+Pro archive.
+
+```yaml
+# chisel.yaml
+format: v1
+archives:
+  <name>:
+    pro: <supported-pro-value>
+    ...
+```
+
+```{include} /reference/chisel-releases/chisel.yaml.md
+  :start-after: <!-- Start: Supported Pro values -->
+  :end-before:  <!-- End: Supported Pro values -->
+```
+
+### Configuring multiple archives
+
+You may define multiple archives. Use the
+{ref}`chisel_yaml_format_spec_archives_priority` field to specify archive
+priorities.
+
+```{tip}
+```{include} /reference/chisel-releases/chisel.yaml.md
+  :start-after: <!-- Start: Suggested Priority values -->
+  :end-before:  <!-- End: Suggested Priority values -->
+```
+
+
+(install_pro_packages_configure_credentials)=
+
```

Comment:
I am re-reading , and it just doesn't sit well with me, for a guide. I.e. the user doesn't need to do anything here. You're "explaining" how it work, but not really guiding the user into doing anything.

Should this be an explanation?

Reply:
Hmmm, you might have a good point here. @yhontyk what do you think?

Reply:
I think @cjdcordeiro was talking about moving the whole guide as an explanation. In any case, I have pushed 527ce6f per your comments. Let me know what you think @cjdcordeiro, @yhontyk.